index.html

```diff
@@ -122,6 +122,10 @@
       This specification is implemented at least by the <a href="http://www.thingweb.io/">Eclipse Thingweb</a>
       project also known as <a href="https://github.com/eclipse/thingweb.node-wot">node-wot</a>, which is considered the reference open source implementation at the moment. Check its <a href="https://github.com/eclipse/thingweb.node-wot"> source code</a>, including <a href="https://github.com/eclipse/thingweb.node-wot/examples">examples</a>.
     </p>
+    <p class="ednote" title="WoT Interaction vs Interaction Affordance">
+      <a>Interaction Affordance</a> it is the correct term when referring Thing capabilities (as explain <a href="https://github.com/w3c/wot-thing-description/issues/282">here</a>). 
```

Comment:
explain_ed_

Reply:
Trying to reformulate slightly, what about
```
<a>Interaction Affordance</a> (or shortly, affordance) it is the term used in [[!WOT-TD]] 
when referring to <a>Thing</a> capabilities, as explained in 
<a href="https://github.com/w3c/wot-thing-description/issues/282">TD issue 282</a>). 
However, this term is not well understood outside the <a>TD</a> semantic context. 
Hence for the sake of the readability, this document will use the previous term 
<a>WoT interaction</a> or, simply, <em>interaction</em> instead. 
```

Reply:
Nice! it is better. Another question: should we move this motivation to the WoT interaction definition? it would be easier to find if, for example, the reader doesn't read the document linearly.

Reply:
Makes sense.